### ISSUE TITLE
Call to undefined method PhpAmqpLib\Message\AMQPMessage::ack()

### ISSUE THREAD

**Issue by slloyd88** [php-amqplib/php-amqplib#1173]
seems ack is missing in message
PHP Fatal error:  Uncaught Error: Call to undefined method PhpAmqpLib\Message\AMQPMessage::ack()
```
<?php
require_once 'c:/wasql/vendor/autoload.php';
use PhpAmqpLib\Connection\AMQPStreamConnection;
use PhpAmqpLib\Exchange\AMQPExchangeType;
use PhpAmqpLib\Message\AMQPMessage;
use PhpAmqpLib\Wire\AMQPTable;

$queue = 'rmqodhjspdq';

$connection = new AMQPStreamConnection('*******', 5673, '*******', '*******', '*******');
$channel = $connection->channel();
$channel->queue_declare($queue, false, true, false, false, false, new AMQPTable(['x-queue-type' => 'quorum']));
$message = $channel->basic_get($queue);
$message->ack();

var_dump($message->body);

$channel->close();
$connection->close();
```

**Comment by andrew-demb** [comment]
https://github.com/php-amqplib/php-amqplib/blob/cb514530ce45a6d2f636be5196010c47c3bcf6e0/PhpAmqpLib/Message/AMQPMessage.php#L106

**Comment by ramunasd** [comment]
Method `ack()` is available since version 2.12
Please make sure You have latest library version.
